0012-Pulse-Compiler-and-IR.md

```diff
@@ -1,6 +1,6 @@
 # Pulse Compiler & IR
 
-| **Status**        | **Proposed** |
+| **Status**        | **Deprecated** |
```

Comment:
Maybe call this "Not planned" or "Not planned anymore"? "Deprecated" risks it sounding like these mechanisms will be removed entirely.